documentation/wiki/Mac-Debugging.md

```diff
@@ -0,0 +1,38 @@
+#Debugging with MacOS
+Say unit tests are failing from Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64
+* Open terminal 
+  * Hit command-space, type terminal, hit enter. Alternatively, hit magnifying glass (spotlight) in upper-right corner and search for terminal.)
+* Build something
+  * Navigation in terminal is similar to command prompt (cd), although you type `ls` in place of `dir`.
+  * **Use ./build.sh instead of .\build.cmd.**
+* Type `find . -name Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64.log`
+  * This should print out a path (from your current working directory) to the relevant log file.
+* Type `tail <path from previous step>`
+  * This prints out the last part of that file.
+  * You can also just open it normally from a finder window.
+  * This file contains the standard output from the last run.
+  * You may notice that the file ends with `=== COMMAND LINE ===` followed by a single (long) command line statement.
```

Comment:
If you take the `tail -1` edit above, the command line will be the only output so this becomes a factoid about opening the whole file (still valuable) instead of critical.

Reply:
That doesn't seem to be an option?

Nathans-MacBook-Pro:msbuild nathanmytelka$ tail -l ./artifacts/log/Debug/Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64.log
tail: illegal option -- l
usage: tail [-F | -f | -r] [-q] [-b # | -c # | -n #] [file ...]